src/app/story/directives/status.component.ts

```diff
@@ -156,6 +158,8 @@ export class StoryStatusComponent implements DoCheck {
   togglePublish() {
     this.isPublishing = true;
     this.story.setPublished(!this.story.publishedAt).subscribe(() => {
+      this.angulartics2.eventTrack.next({ action: this.story.publishedAt ? 'publish' : 'unpublish',
+        properties: { category: 'episode', label: 'episode/' + this.story.doc.id }});
```

Comment:
can't save in a local variable and use in all three places cuz it changes with the publish/unpublish. gonna keep using the date as boolean because it seems like there are some pretty strong feelings about that.